frontend/dashboard/components/RepoList/ActionLinks.tsx

```diff
@@ -0,0 +1,96 @@
+import { Button, DropdownMenu } from '@digdir/design-system-react';
+import React, { useRef, useState } from 'react';
+import classes from './RepoList.module.css';
+import cn from 'classnames';
+import {
+  ExternalLinkIcon,
+  FilesIcon,
+  MenuElipsisVerticalIcon,
+  PencilIcon,
+} from '@navikt/aksel-icons';
+import { useTranslation } from 'react-i18next';
+import { getRepoEditUrl } from '../../utils/urlUtils';
+import type { Repository } from 'app-shared/types/Repository';
+import { MakeCopyModal } from '../MakeCopyModal';
+
+type ActionLinksProps = {
+  repo: Repository;
+};
+
+export const ActionLinks = ({ repo }: ActionLinksProps): React.ReactElement => {
+  const { t } = useTranslation();
+  const [copyCurrentRepoName, setCopyCurrentRepoName] = useState('');
+  const [modalOpen, setModalOpen] = useState<boolean>(false);
+  const copyModalAnchorRef = useRef(null);
+
+  const handleOpenCopyModal = (repoFullName: string) => {
+    setModalOpen(true);
+    setCopyCurrentRepoName(repoFullName);
+  };
+
+  const handleCloseCopyModal = () => {
+    setModalOpen(false);
+    setCopyCurrentRepoName(null);
+  };
+
+  const repoFullName = repo.full_name;
+  const [org, repoName] = repoFullName.split('/');
+  const isDatamodelling = repoFullName.endsWith('-datamodels');
```

Comment:
A general comment for the whole dashboard; should we set a global const for the dataModelRepoIdentifyer? 🤔 So it is clear where the identifyer is used.

If it is a large task to do so, we can create a new issue?

Reply:
Good call! 😄 
It was only used a handful of places in the dashboard. Fixed ✅ 

```
export const DATA_MODEL_REPO_IDENTIFIER = '-datamodels';
```